backend/sirius-web-emf/src/main/java/org/eclipse/sirius/web/emf/services/EditService.java

```diff
@@ -202,23 +211,37 @@ public void delete(Object object) {
     }
 
     @Override
-    public List<Namespace> getNamespaces() {
+    public List<Namespace> getNamespaces(UUID editingContextId) {
+        List<String> eNsURIs = new ArrayList<>();
```

Comment:
In [this patch](https://github.com/eclipse-sirius/sirius-components/pull/394/commits/095d773a66eef7c51e849bba2efdfde8ab6153a3#diff-e7d3e5ee9279360b18929f33685787e6c6bd71059f15c112b996fa80ee739f68R63) I used a `Map<UUID, IRepresentationDescription> allRepresentationDescriptions = new LinkedHashMap<>();` (then get its `value()`) to support the case where a dynamically contributed element with the same key as a statically known one wins. In my case the key is the `UUID`, here it is the `nsURI` but the logic is similar: using a `List<String>` here allows for duplicate `nsURIs` and namespaces.